src/main/java/com/toptrumps/cli/CommandLineUI.java

```diff
@@ -40,8 +48,17 @@ public void startGame() {
         logger.printToLog("New line \n new line too.");
         input = scanner.nextLine();
         if (input.equalsIgnoreCase("f")) {
+            user = new Player("Human");
```

Comment:
I'd reconsider this design decision. We already know that there's only one Human Player and this logic applies to both Apps. The Game object was supposed to encapsulate all the game logic, including "Player".